src/Chat.tsx

```diff
@@ -254,3 +282,15 @@ const ResizeDetector = (props: {
                 frame.contentWindow.onresize = props.onresize;
         } }
     />;
+
```

Comment:
Can you add a comment as to these special keys

Reply:
Done.